src/parser/UpdateClause.h

```diff
@@ -0,0 +1,21 @@
+//  Copyright 2024, University of Freiburg,
+//  Chair of Algorithms and Data Structures.
+//  Author: Julian Mundhahs (mundhahj@informatik.uni-freiburg.de)
+
+#pragma once
+
+#include "parser/SelectClause.h"
+#include "parser/Triples.h"
+#include "parser/data/Types.h"
+
+namespace parsedQuery {
+struct UpdateClause : ClauseBase {
+  std::vector<SparqlTripleSimple> toInsert_;
+  std::vector<SparqlTripleSimple> toDelete_;
+
+  UpdateClause() = default;
+  explicit UpdateClause(std::vector<SparqlTripleSimple> toInsert,
```

Comment:
no need for `explicit` here, it prevents e.g. the pattern `return{toInsert, toDelete}` in a function with return type `UpdateClause`